index.html

```diff
@@ -1127,8 +1127,10 @@ <h2>Example</h2>
 
 <section id="errors">
 	<h1>Errors</h1>
-	<p class="issue">Do we need to define a list of error conditions, codes, etc.</p>
-
+	<h2>internalError</h2>
+    <p>When an unexpected error occurs during did resolution or dereferencing 
+    the resolution or dereferencing meta data error should be <i>internalError</i> and status code 500 
```

Comment:
Got merged too quick...  Needs 2 commas, as shown --
```suggestion
    <p>When an unexpected error occurs during did resolution or dereferencing, 
    the resolution or dereferencing meta data error should be <i>internalError</i>, and status code 500 
```

Reply:
Sorry I should have waited longer. I now applied your suggestion in https://github.com/w3c-ccg/did-resolution/commit/746b326570895fc6ce014a659e5d9d55ba9efec0.